core/domain/classifier_services.py

```diff
@@ -127,6 +127,69 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+
+
+def add_to_training_queue(exploration):
```

Comment:
@seanlip If thats the case, I don't really mind. You're method of doing this looks very neat too. I'll re-structure it accordingly then.